x-pack/plugins/security_solution/server/endpoint/utils/action_list_helpers.ts

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
I did not see a test file for this module - will you still include it? or defer to a later time?

Reply:
Yeah I'll add tests for the helpers